react/slider/stories/Slider.stories.tsx

```diff
@@ -0,0 +1,491 @@
+import * as React from 'react';
+
+import { Slider } from '../src/index.ts';
+
+const metadata = { title: 'Forms/Slider' };
+export default metadata;
+
+export const Regular = () => {
+  const [value, setValue] = React.useState(20000);
+
+  return (
+    <div>
+      <output>{value}</output>
+      <Slider onChange={(value) => setValue(value)} value={value} min={10000} max={60000} step={10} />
+      <button onClick={() => setValue(15000)}>Reset</button>
+    </div>
+  );
+};
+
+export const Regular1 = () => {
```

Comment:
Maybe give a proper name to each story, Like why this story is different from the previous one?